### DISCUSSION TITLE
Homes support for Bungeecord

### DISCUSSION THREAD

**Discussion by RSLuke** [Zrips/CMI#2354] [Feature Requests]
**Description of feature/improvement request:**

Be able to teleport to homes between servers.

---

### Basic explanation as to why this feature is needed and/or useful to have: ###

 I have several servers with the same database of CMI, the players can switch between servers to access other world so I can have more players. Problem is I have to simulate fake worlds on every server so the homes won't delete.

**Comment by LoToRTv** [comment]
+1
